automatic/tor-browser/tools/chocolateyInstall.ps1

```diff
@@ -8,10 +8,10 @@ $destinationFolder = GetInstallDirectory -toolsPath $toolsDir
 $packageArgs = @{
   PackageName  = 'tor-browser'
   FileType     = 'exe'
-  Url          = 'https://www.torproject.org/dist/torbrowser/12.5.6/torbrowser-install-12.5.6_ALL.exe'
```

Comment:
Understood, I didn't revert this file back to v12.5.6 since the update script will overwrite it anyway. Should I revert back to the original 12.5.6 URL/hashes?

Reply:
Gotcha, will do this tonight.

Reply:
@TheCakeIsNaOH, done! Thanks for the guidance.